jupyter-notebooks/analysis-ready-data/ard_1_intro_and_best_practices.ipynb

```diff
@@ -614,8 +649,6 @@
    "source": [
     "#### Step 4: Download Order\n",
     "\n",
-    "To download the order from the orders API, we will use the planet python client CLI. It would be nice to use the python client python API for this step but, as of the writing of this tutorial, support for the orders API in the planet client python API has been [confusing](https://github.com/planetlabs/planet-client-python/issues/217). The CLI download status output is also [confusing and possibly wrong](https://github.com/planetlabs/planet-client-python/issues/218) but the CLI does easily and successfully download the order.\n",
```

Comment:
May I suggest one small edit here?

Let's mention that we're using the Orders API via the CLI to download the order. Jumping straight into the manifest file without mentioning how we're downloading the data feels incomplete.